### PR TITLE
Mark Quartz extension as stable

### PR DESCRIPTION
**Pull request by gastaldi**
This promotes the extension status from preview to stable

### CODE REVIEW

**Comment by gsmet** [comment]
Probably a good idea to check with @mkouba before promoting it.

**Comment by geoand** [comment]
Makes sense to me, but let's let @mkouba and @machi1990 weigh in here

File: extensions/quartz/runtime/src/main/resources/META-INF/quarkus-extension.yaml [hunk 1]
```diff
@@ -10,6 +10,6 @@ metadata:
   guide: "https://quarkus.io/guides/quartz"
   categories:
   - "miscellaneous"
-  status: "preview"
+  status: "stable"
   config:
   - "quarkus.quartz."
```
Comment (machi1990) [inline extensions/quartz/runtime/src/main/resources/META-INF/quarkus-extension.yaml:15]:
```suggestion
  - "quarkus.quartz."

```

**Review by machi1990 — DISMISSED** [review]
+1 on my side.

The API has been stable for quite some time now. 

On top of the change in the extensin config yaml, we'll also need to remove the preview status from the doc: https://github.com/quarkusio/quarkus/blob/main/docs/src/main/asciidoc/quartz.adoc.

**Review by gsmet — DISMISSED** [review]

**Comment by gsmet** [comment]
@mkouba we waited for you on this one :).

**Comment by mkouba** [comment]
> @mkouba we waited for you on this one :).

@gsmet I'm personally +0 on this one. It's true that our API can be considered stable but there are some Quartz features which are not extensively tested (such as clustering) and thus our support is "limited".

OTOH if @machi1990 thinks that it's stable then I'm in as well :-)

**Review by gsmet — APPROVED** [review]
OK, let's proceed then.

**Pull request merged**